atomata/capstone_WebClient
# Adding Styled Scroller for Action Sequence list

### Summary 
This PR removes the previously expendable action items, and adds a scrollbar to allow user to navigate longer action sequence lists
### How to test?
1. Spin up the App
2. Login and choose an apparatus and enter the editor
3. When no actions are in action sequence, the space should be empty
4, When actions are added to the action sequence list, the scrollbar becomes visible
### Checks
